homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -129,6 +130,14 @@ def __init__(
         self.entity_description = description
         self._attr_unique_id = f"{service[SERVICE_ID]}:{description.key}"
         self._attr_name = f"{service['name']} {description.name}"
+        self._attr_device_info = DeviceInfo(
+            entry_type=DeviceEntryType.SERVICE,
```

Comment:
You need to  import `DeviceEntryType` as well

Reply:
Yep, accepted the suggestion on my phone, will fire up my dev environment to fix this shortly.